ast/ddl.go

```diff
@@ -194,6 +195,34 @@ func (n *DropDatabaseStmt) Accept(v Visitor) (Node, bool) {
 	return v.Leave(n)
 }
 
+// RenameDatabaseStmt is a statement to rename a database.
+// It supports in MySQL 5.1.7 and removed in MySQL 5.1.23.
+type RenameDatabaseStmt struct {
+	ddlNode
+
+	OldDB string
```

Comment:
s/string/CIStr

Reply:
Why do we need to do this?

Reply:
@zimulala If not do this, we need convert it to `CIStr` when use this field in TiDB, e.g.
`model.NewCIstr(renameDatabaseStmt.OldDB)`.
If change to `CIStr` here, we can use it directly in TiDB.

Reply:
I think it's OK. "create database" and "drop database" all use `string`.